dnf-behave-tests/features/repoquery.feature

```diff
@@ -153,3 +153,12 @@ Given I use the repository "dnf-ci-thirdparty"
  .+/fixtures/repos/dnf-ci-thirdparty/src/SuperRipper-1.0-1.src.rpm
  .+/fixtures/repos/dnf-ci-thirdparty/x86_64/SuperRipper-1.0-1.x86_64.rpm
  """
+
+
+Scenario: dnf repoquery --querytags is working
+ When I execute dnf with args "repoquery --querytags"
+ Then the exit code is 0
+ And stdout contains lines
+ """
+ Available query-tags: use --queryformat ".. %{tag} .."
+ """
```

Comment:
How about testing the exact match in this case?
The querytags are always printed in the same order and the lines are not wrapped with smaller terminal width, so it shouldn't be a problem.

"""
Available query-tags: use --queryformat ".. %{tag} .."

name, arch, epoch, version, release, reponame (repoid), evr,
debug_name, source_name, source_debug_name,
installtime, buildtime, size, downloadsize, installsize,
provides, requires, obsoletes, conflicts, sourcerpm,
description, summary, license, url

"""

Reply:
No problem, done.